lib/src/dartdoc_analyzer.dart

```diff
@@ -28,3 +32,37 @@ Suggestion getDartdocRunFailedSuggestion([DartdocResult result]) {
     score: 10.0,
   );
 }
+
+/// Creates a report section about documentation coverage.
+/// 20% coverage grants the maximum number of points.
+ReportSection documentationCoverageSection({
+  @required int documented,
+  @required int total,
+}) {
+  final percent = total <= 0 ? 1.0 : documented / total;
+  final accepted = percent >= 0.2;
+  final granted = accepted ? 10 : 0;
+  final undocumented = total - documented;
+  return ReportSection(
+    title: documentationSectionTitle,
+    grantedPoints: granted,
+    maxPoints: 10,
+    summary: accepted
+        ? 'At least 20% of the public API has documentation comments.'
```

Comment:
I'd like the first line to be invariant to whether the check passed or not, just explaining the scoring. Then a paragraph explaining the eventual problems.

In this case it would probably make sense to print the percentage even if it's above 20%.

Reply:
PTAL.